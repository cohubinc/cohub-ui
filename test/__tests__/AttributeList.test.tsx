import React from "react";
import { shallow } from "enzyme";

import { AttributeList } from "dist";

it("renders without crashing", async () => {
  const wrapper = shallow(
    <AttributeList
      header={"test"}
      items={[{ attribute: "test", value: "test", format: "text" }]}
    />
  );

  expect(wrapper).toMatchSnapshot();
});
