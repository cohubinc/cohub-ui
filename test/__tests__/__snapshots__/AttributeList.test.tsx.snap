// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Segment
  bordered={false}
  className="flex flex-column"
  elevation={0}
  padded={false}
>
  <Component
    className="mb-1"
    muted={true}
    uppercase={true}
    weight={500}
  >
    test
  </Component>
  <div
    className="flex justify-between items-center my-05 undefined"
    key="test"
  >
    <Typography>
      test
    </Typography>
    <Typography
      alignment="right"
    >
      test
    </Typography>
  </div>
</Segment>
`;
